.vscode/launch.json

```diff
@@ -28,6 +36,7 @@
             "args": ["migrate", "up", "0"],
             "env": {
                 "CONFIG_PATH": "${workspaceFolder}/configs",
+                "ENV_NAME": "local",
```

Comment:
This is removed with the refactor at https://github.com/podengo-project/idmsvc-backend/pull/178